x-pack/plugins/security_solution/public/timelines/components/timeline/header/translations.ts

```diff
@@ -21,3 +22,66 @@ export const CALL_OUT_IMMUTABLE = i18n.translate(
       'This prebuilt timeline template cannot be modified. To make changes, please duplicate this template and make modifications to the duplicate template.',
   }
 );
+
+export const EDIT = i18n.translate('xpack.securitySolution.timeline.saveTimeline.modal.button', {
+  defaultMessage: 'edit',
+});
+
+export const SAVE_TIMELINE = i18n.translate(
+  'xpack.securitySolution.timeline.saveTimeline.modal.header',
+  {
+    defaultMessage: 'Save Timeline',
+  }
+);
+
+export const SAVE_TIMELINE_TEMPLATE = i18n.translate(
+  'xpack.securitySolution.timeline.saveTimelineTemplate.modal.header',
+  {
+    defaultMessage: 'Save Timeline Template',
+  }
+);
+
+export const SAVE = i18n.translate('xpack.securitySolution.timeline.nameTimeline.save.title', {
+  defaultMessage: 'Save',
+});
+
+export const NAME_TIMELINE = i18n.translate(
+  'xpack.securitySolution.timeline.nameTimeline.modal.header',
+  {
+    defaultMessage: 'Name Timeline',
+  }
+);
+
+export const NAME_TIMELINE_TEMPLATE = i18n.translate(
+  'xpack.securitySolution.timeline.nameTimelineTemplate.modal.header',
+  {
+    defaultMessage: 'Name Timeline Template',
+  }
+);
+
+export const DISCARD_TIMELINE = i18n.translate(
+  'xpack.securitySolution.timeline.saveTimeline.modal.discard.title',
+  {
+    defaultMessage: 'Discard Timeline',
+  }
+);
+
+export const DISCARD_TIMELINE_TEMPLATE = i18n.translate(
+  'xpack.securitySolution.timeline.saveTimelineTemplate.modal.discard.title',
+  {
+    defaultMessage: 'Discard Timeline Template',
+  }
+);
+
+export const CLOSE_MODAL = i18n.translate(
+  'xpack.securitySolution.timeline.saveTimeline.modal.close.title',
+  {
+    defaultMessage: 'Close',
+  }
+);
+
+export const UNSAVED_TIMELINE_WARNING = (timelineType: TimelineTypeLiteral) =>
+  i18n.translate('xpack.securitySolution.timeline.saveTimeline.modal.warning.title', {
+    values: { timeline: timelineType === TimelineType.template ? 'timeline template' : 'timeline' },
+    defaultMessage: 'You have an unsaved {timeline}, do you wish to save it?',
```

Comment:
Sorry, it looks like the wording I suggested here is an example of a [comma slice](https://www.grammarly.com/blog/comma-splice/), where two independent clauses are joined by a comma.

Would you be willing to change it to the following?

```
'You have an unsaved {timeline}. Do you wish to save it?'
```